master/buildbot/buildslave/ec2.py

```diff
@@ -149,6 +149,10 @@ def __init__(self, name, password, instance_type, ami=None,
                 self.conn = boto.ec2.connect_to_region(region,
                                                        aws_access_key_id=identifier,
                                                        aws_secret_access_key=secret_identifier)
+                if subnet_id is not None:
+                    self.vpc_api_conn = boto.vpc.VPCConnection(region=region_found,
+                                                           aws_access_key_id=identifier,
```

Comment:
Please align arguments with opening brace (as done in other places in this file).